C/README.md

```diff
@@ -114,6 +114,7 @@ _add list here_
 ## Mathematics
 
 - [Binary Exponentiation](math/Binary_Exponentiation.c)
+- [Binary to Decimal](math/binary_to_decimal.c)
```

Comment:
Rewrite it as `- [Binary to Decimal Conversion] `

Shift the problem to other file location as this is not a math problem.

Reply:
done sir, please review.